x/gov/keeper/proposal.go

```diff
@@ -134,6 +134,14 @@ func (k Keeper) SubmitProposal(ctx sdk.Context, content govtypes.Content, addr s
 		return types.Proposal{}, err
 	}
 
+	if k.IsCouncilMember(ctx, addr) {
```

Comment:
I don't think proposal status should be modified here as this is being done in the method `ActivateCouncilProposalVotingPeriod()`.

Reply:
If I remove this, the status for [test](https://github.com/priyanshiiit/shentu/blob/c1e8d2dfa67e2ebf17130569318724e79b63e989/x/gov/keeper/grpc_query_test.go#L91-L96) remains 1 and the test fails.

Reply:
We need to fix the test. We only want certifier voting rounds for certian types of proposals.